components/Paper/Tabs/DiscussionTab.js

```diff
@@ -7,19 +7,23 @@ import Ripples from "react-ripples";
 import ReactPlaceholder from "react-placeholder";
```

Comment:
Thanks for starting to refactor this. 
I've already created ticket for this for both FE & BE. 
But let's really consider migrating away from the old commenting modules.
Anytime, this component / BE needs to be updated for these, it's too much effort & I don't think it's going to be scalable the way we have them.